openvdb_houdini/houdini/GEO_VDBTranslator.cc

```diff
@@ -231,14 +236,11 @@ GEO_VDBTranslator::fileStat(const char *filename, GA_Stat &stat, uint level)
 
     return true;
 }
+#endif
```

Comment:
This endif is un-paired, I've added a pull request into sideeffects/send_upstream that should fix this:

https://github.com/sideeffects/openvdb_dev/pull/1